test/fixtures/hdfs-fixture/build.gradle

```diff
@@ -18,25 +18,22 @@
  */
 
 apply plugin: 'elasticsearch.build'
+apply plugin: 'elasticsearch.test.fixtures'
 
-versions << [
-  'hadoop2': '2.8.1'
-]
-
-// we create MiniHdfsCluster with the hadoop artifact
 dependencies {
-  compile "org.apache.hadoop:hadoop-minicluster:${versions.hadoop2}"
+  compile "org.apache.hadoop:hadoop-minicluster:2.8.1"
+}
+
+task syncClasses(type: Sync) {
+  from sourceSets.test.compileClasspath
```

Comment:
It's not completely clear to me what we are using this for. Should we really be using `runtimeClasspath` here? If so we can actually just use that and omit the explicitly dependency sourceset output as that's included in the runtime classpath.